test/prototype/test_transforms.py

```diff
@@ -256,3 +257,26 @@ def test_mask_transform_mask_bos(self):
             masked_tokens, _, _ = mask_transform(self.sample_token_ids)
             exp_tokens = torch.tensor([[5, 5, 5, 5, 5], [5, 5, 5, 4, 4]])
             self.assertEqual(exp_tokens, masked_tokens)
+
+    def _t5tokenizer(self, test_scripting):
+        asset_name = "https://huggingface.co/t5-base/resolve/main/spiece.model"
```

Comment:
We don't try not to rely on any network downloads for our unit tests. For other tests related to the SPM tokenizer, you will see we rely on an `spm_example.model` file contained in the assets directory. Can we use the same asset when testing our T5Transform?

Reply:
sorry, left this from my own local tests. will update